### ISSUE TITLE
XFGloss incompatible with default Xamarin.Forms setup in Visual Studio 2017

### ISSUE THREAD

**Issue by b9chris** [tbaggett/xfgloss#37]
1) Create a new Xamarin.Forms UWP/mobile app with all the defaults selected.
2) Install XFGloss

XFGloss appears to install but fails to ever load, indicated by an insidious warning icon on the Nuget Package:
https://imgur.com/a/m8WSSx3

A separate bug, in Visual Studio, prevents getting any more info about what went wrong - but the discussion in this StackOverflow post about the bug implies it's something wrong with how the Nuget Package sets its paths:
https://stackoverflow.com/questions/37208853/net-2015-references-with-yellow-triangle-for-nuget-packages-on-portable-librari

Attempting to use XFGloss in this scenario results in no typeahead on its namespace and build errors if you try to invoke it manually in codebehind or xaml forms.

**Comment by tbaggett** [comment]
First, please be aware that XFGloss doesn't currently support the UWP platform. Only Android and iOS are currently supported.

I am unable to reproduce your issue. I updated to the latest Visual Studio 2017 Community Edition (15.8.7) and created a new "Mobile App (Xamarin.Forms)" project that targeted Android, iOS and UWP using the latest stable Xamarin.Forms release (3.3.0.912540). 

I added XFGloss 1.1.3 to the cross-platform project and all three platform specific projects. I saw no errors after adding it or building the project, as evidenced by the solution explorer screen shot seen at https://imgur.com/a/kRweAyN.

Can you provide the version number of VS 2017 you're using? I saw you were using Xamarin.Forms 3.1 in the screen shot, but me using the latest version shouldn't make a difference, as XFGloss has been used with a variety of versions of XF without issue. XFGloss v1.1.3 requires XF version 2.5.0.280555 or newer.

**Comment by b9chris** [comment]
Thank you, I uninstalled trying to get a feature out, but my suspicion is
that this is solely a bug in Visual Studio and not yours, although there
may be a way to work around it on your end with a precaution of some sort.
Specifically, I've installed a number of Nuget packages without issue, but,
2 now had identical behavior. Yours, and a day after I submitted this,
Xamarin.Essentials. I dug farther since the warning icon and failure to
pick it up was JUST like what I had with your very different Nuget package,
and the vague suggestion was that Visual Studio itself broke. To fix it? I
closed Visual Studio, and opened it back up again.

So I very highly suspect the solution is the same here. It appears to be a
problem with VS2017 Xamarin projects and some category of Nuget packages.
But it's resolved as soon as you close and reopen Visual Studio.
